### ISSUE TITLE
Contribute to nb-git-open-in-external-repoviewer ?

### ISSUE THREAD

**Issue by markieww** [jglick/nbgittricks#1]
Hi Jesse. If you like you can contribute to https://github.com/markiewb/nb-git-open-in-external-repoviewer, which does the same and more even for other hosters. As far as I see you added ranges for viewing a file.

BR Benno

**Comment by Chris2011** [comment]
Sounds great :).

**Comment by jglick** [comment]
Yeah that would make sense. I tried 1.6.0 in a NB dev build from a couple months back and it did not seem to do anything (no menu items offered in editor tab), not sure why yet.
